core/sup/Makefile

```diff
@@ -4,7 +4,7 @@ PROJECT = sup
 ESCRIPT = sup
 CLEAN_MOAR = clean-escript
 
-all: compile $(ESCRIPT)
+all: compile #$(ESCRIPT)
```

Comment:
Why are we commenting out escript?   This builds the `sup` tool that is used, we absolutely need it.